devtools/gqltsgen/run.go

```diff
@@ -5,28 +5,36 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"path/filepath"
+	"slices"
 
+	"github.com/vektah/gqlparser/v2"
 	"github.com/vektah/gqlparser/v2/ast"
-	"github.com/vektah/gqlparser/v2/parser"
 )
 
 func main() {
 	out := flag.String("out", "", "Output file.")
 	flag.Parse()
 	log.SetFlags(log.Lshortfile)
 	var src []*ast.Source
-	for _, file := range flag.Args() {
-		data, err := os.ReadFile(file)
+	for _, pattern := range flag.Args() {
+		files, err := filepath.Glob(pattern)
 		if err != nil {
 			log.Fatal("ERROR:", err)
 		}
-		src = append(src, &ast.Source{
-			Name:  file,
-			Input: string(data),
-		})
+		for _, file := range files {
+			data, err := os.ReadFile(file)
+			if err != nil {
+				log.Fatal("ERROR:", err)
+			}
+			src = append(src, &ast.Source{
+				Name:  file,
+				Input: string(data),
+			})
+		}
 	}
 
-	doc, err := parser.ParseSchemas(src...)
+	sch, err := gqlparser.LoadSchema(src...)
```

Comment:
LoadSchema interprets things like `extend type` whereas ParseSchemas keeps it as a document and fails to normalize things like that